### PR TITLE
Working condition in windows system and updated readme

### PR DESCRIPTION
**Pull request by jai-dewani**
- Lowered youtube-dl version
- Added sample.env for easy configuration in new systems

### CODE REVIEW

**Comment by jai-dewani** [comment]
The build is successful on semaphoreci!

File: app.js [hunk 1]
```diff
@@ -37,24 +36,21 @@ app.get("/search/:query", function(req, res) {
...
         res.render("index", {data: data});
     }
 );
 });
 
 app.get("/download/:videoUrl", function(req, res) {
-    console.log(cwd);
-    var video = youtube("https://www.youtube.com/embed/" + req.params.videoUrl, 
-    ["--format=18"], 
+    const video = youtube("http://www.youtube.com/watch?v=" + req.params.videoUrl,
```
Comment (utkarsh-raj) [inline app.js:52]: Let this be var, for now, I suspect problems in asynchronous nature if the variable is constant.

File: package.json [hunk 2]
```diff
@@ -15,7 +15,8 @@
     "express": "^4.16.4",
     "google-auth-library": "^2.0.1",
     "googleapis": "^35.0.0",
+    "nodemon": "^2.0.3",
```
Comment (utkarsh-raj) [inline package.json:18]: Please add nodemon as a dev dependency.

**Comment by jai-dewani** [comment]
Made the changes as requested
Also added a simple readme.md which should also fix #17

**Pull request closed without merging**